src/basis/basis.js

```diff
@@ -1,12 +1,13 @@
 import _ from "underscore";
 import s from "underscore.string";
-import {getElectronegativity} from "@exabyte-io/periodic-table.js";
+import {PERIODIC_TABLE, getElectronegativity} from "@exabyte-io/periodic-table.js";
 
 import math from "../math";
-import {Lattice} from "../lattice/lattice";
+import {Lattice, nonPeriodicLatticeMinimumSizeAngstroms} from "../lattice/lattice";
```

Comment:
I thought this has to be a factor?? The distance is calculated in `maxPairwiseDistance`